src/seth/libexec/seth/seth

```diff
@@ -74,6 +78,9 @@ full                 get block with full transaction information
 
   Options for seth-call(1):
 B,block=number       block number (default: \`latest')
+hevm                 execute with hevm
+debug                execute with hevm interactive mode
+code                 bytecode to replace the called contract with
```

Comment:
state as well, right?

Reply:
Yes. It will already be captured because it is defined in the run-tx opts, I'm just hesitant to document it fully as an option currently.

Reply:
ok, I guess it can be our secret for now

Reply:
It's partly because I think --state should only actually be writeable by `seth send`, where `seth call` would be read-only, but I think it needs a little more thought. We could have a persistent blockchain stored in the filesystem under git, which would be great for learning / experimentation compared to geth/parity db.

Reply:
hmmm, that's an interesting thought. So then --state would cause tx to not get sent at all?

Reply:
I think so yes. --state is only meaningful when using hevm. `seth send --hevm --state dir` would write to the state dir, or discard the tx if the state dir is undefined.

Something like:

- `seth call`: read-only chain, no access to --state
- `seth send`: write chain, no access to --state
- `seth call --hevm`, read-only chain, read-only --state
- `seth send --hevm`, read-only chain, write --state